packages/xo-server-transport-nagios/src/index.js

```diff
@@ -114,16 +144,30 @@ class XoServerNagios {
       {
         message: 'The server-nagios plugin for Xen Orchestra server seems to be working fine, nicely done :)',
         status: OK,
+        isTest: true,
```

Comment:
What's the point of this?

Reply:
It's to handle this specific situation.

But I see in your comment below that I should handle up generally.